index.bs

```diff
@@ -843,6 +844,26 @@ spec:css-syntax-3;
         or [user-switching](#example-mediation-switch) scenarios. Further, the requirement is tied
         to a specific operation, and does not affect the [=origin/prevent silent access flag=] for the
         origin. To set that flag, developers should call {{preventSilentAccess()}}.
+
+    :   <dfn>immediate</dfn>
+    ::  The user agent will not hand over credentials without [=user mediation=].
+        If the user agent determines a lack of applicable credentials, the user agent will throw a "{{NotAllowedError}}" {{DOMException}} without [=user mediation=].
```

Comment:
Similarly, this needs to be in an algorithm or more explicitly linked to an algorithm.

Reply:
Removed. This is specified in the `DiscoverFromExternalSources` method of `PublicKeyCredential`.